js/navbar.js

```diff
@@ -1,3 +1,19 @@
-document.addEventListener('DOMContentLoaded', function () {
-    console.log('ready1');
-});
+$(document).ready(function() {
+	var NavY = $('.rectangle').offset().top;
```

Comment:
tutaj wrzucamy: 
 $.scrollTo(0);
$('.nav a').click(function() { $.scrollTo($(this).attr('href'), 500); });